src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+    private static String[] tasks = new String[100];
```

Comment:
I like how you used "tasks"  for the name of a collection of objects. Good work!